pages/Messages/Overview.md

```diff
@@ -112,61 +78,19 @@ A public reply post.
 | ------------- |------------- | ---- | --- |
 | inReplyTo | ID of the announcement the reply references |  bytes32 | YES
```

Comment:
I think these inReplyTos are incorrect now. They should be string style identifiers: https://spec.projectliberty.io/Messages/Identifiers